src/java.base/share/classes/sun/security/ssl/SSLTrafficKeyDerivation.java

```diff
@@ -154,8 +154,8 @@ public SecretKey deriveKey(String algorithm,
                         ks.getKeyLength(cs),
                         ks.getAlgorithm(cs, algorithm));
             } catch (GeneralSecurityException gse) {
-                throw (SSLHandshakeException)(new SSLHandshakeException(
-                    "Could not generate secret").initCause(gse));
+                throw new SSLHandshakeException(
+                        "Could not generate secret", gse);
```

Comment:
I can't quite tell given the coloring, but did this get changed into a tab?

Reply:
Yes, I added 4 more whitespaces in line 158.

Reply:
Ah, missed that.  Good to be consistent.